src/systemctl/systemctl.c

```diff
@@ -4275,13 +4286,18 @@ static void print_status_info(
         }
 
         ss = streq_ptr(i->active_state, i->sub_state) ? NULL : i->sub_state;
-        if (ss)
-                printf("     Active: %s%s (%s)%s",
-                       active_on, strna(i->active_state), ss, active_off);
-        else
+        if (ss) {
+                if (!streq(i->freezer_state, "running"))
+                        printf("   Active: %s%s (%s) (%s)%s",
+                               active_on, strna(i->active_state), ss, i->freezer_state, active_off);
+                else
+                        printf("   Active: %s%s (%s)%s",
+                               active_on, strna(i->active_state), ss, active_off);
```

Comment:
Maybe make "(frozen)" yellow or orange to make it stand out more?

Reply:
Done